src/players/random.py

```diff
@@ -38,5 +38,7 @@ def placeBets(self) -> None:
         Updates the :py:class:`~table.Table` object with a randomly placed :py:class:`~bet.Bet`
         instance.
         """
-
-        self.table.placeBet(Bet(1, self.rng.choice(list(self.all_OC))))
+        if super().playing() and self.stake > 0:
```

Comment:
need to provide a `playing` method. If checked from inside `placeBets`, the sessions will always run all cycles.

Reply:
Done.